src/index.ts

```diff
@@ -442,7 +442,8 @@ export function graphqlHTTP(options: Options): Middleware {
 
     if (isAsyncIterable(executeResult)) {
       response.setHeader('Content-Type', 'multipart/mixed; boundary="-"');
-      sendPartialResponse(pretty, response, formattedResult);
+      // There will always be a next, either a data payload, or an error
```

Comment:
maybe you can help me confirm this? What if no result? Maybe we need to `.next()` and check for the `done`?